cypress.config.ts

```diff
@@ -48,16 +48,21 @@ export default defineConfig({
     },
     defaultCommandTimeout: 8000,
     e2e: {
-        testIsolation: false,
         specPattern: "cypress/e2e/**/*.test.{js,jsx,ts,tsx}",
+        baseUrl: "https://tackle-konveyor-tackle.apps.mtv03.rhos-psi.cnv-qe.rhood.us",
```

Comment:
Note for myself: I believe I tried to replace `tackleUrl` with `baseUrl` and there were some problems, same with `testIsolation`. 

I'll check that.

Reply:
The baseUrl change seems to be working for the most part, I didn't switch most calls to the relative paths, though, most things still use the full url so that might be a followup,

Test Isolation seems to work if you fix up the tests; for the most part, it just means that every test has to have a good starting point with a `cy.visit()` call because otherwise there is no page loaded.